src/main.ts

```diff
@@ -106,7 +106,7 @@ window.onload = function () {
   const sizeSetDom = document.querySelector<HTMLDivElement>('.menu-item__size')!
   const sizeSelectDom = sizeSetDom.querySelector<HTMLDivElement>('.select')!
   const sizeOptionDom = sizeSetDom.querySelector<HTMLDivElement>('.options')!
-  sizeSetDom.title = `设置字号`
+  sizeSetDom.title = `set font size`
```

Comment:
let's not update main.ts as well, since the changes in this file are not relevant for our app.

Reply:
Sure.